lib/coordinates.rb

```diff
@@ -0,0 +1,59 @@
+class Coordinates
+  def initialize(coords)
+    @coords = coords.split(" ")
+  end
+
+  def are_diagonal?
+    not are_vertical? and not are_horizontal?
+  end
+
+  def are_vertical?
+    expected_length = @coords.length
+    y_coords = @coords.map do |coord|
+      coord[1..-1].to_i
+    end
+    y_coords.uniq.length == 1
```

Comment:
Chained methods, but I have no problem. Will the instructors have an issue?

Reply:
Maybe, but it's quite a nitpick imo and we can ask the instructor if this is ok for this instance.